src/measure_theory/vector_measure.lean

```diff
@@ -618,6 +623,90 @@ begin
   { rw [v.not_measurable hi, w.not_measurable hi] }
 end
 
+notation v ` ≤[`:25 i:25 `] `:0 w:25 := v.restrict i ≤ w.restrict i
```

Comment:
I think you'll have a better time with
```suggestion
notation v ` ≤[`:25 i:25 `] `:0 w:25 := vector_measure.restrict v i ≤ vector_measure.restrict w i
```
which should ensure lean knows what `0` means

Reply:
That fixed it, thanks!

Reply:
Edited, you need the `measure_theory.` prefix too.